soh/src/overlays/actors/ovl_En_Cow/z_en_cow.c

```diff
@@ -209,12 +216,129 @@ void func_809DF730(EnCow* this, GlobalContext* globalCtx) {
     }
 }
 
+struct CowInfo {
+    int cowId;
+    RandomizerCheck randomizerCheck;
+};
+
+struct CowInfo EnCow_GetCowInfo(EnCow* this, GlobalContext* globalCtx) {
+    s32 uniqueCoords = this->actor.world.pos.x + this->actor.world.pos.z;
```

Comment:
i still don't like using x+z here, it just seems really odd. we should just check x and z separately